q2_moshpit/kraken2/classification.py

```diff
@@ -75,14 +76,22 @@ def get_paths_for_contigs(contig_id, fp):
         return contig_id.rstrip('_contigs'), [fp]
 
     try:
-        if manifest is not None:  # we got reads - use the manifest
+        if manifest is not None:
+            # we got reads - use the manifest
             iterate_over = manifest.iterrows()
             path_function = get_paths_for_reads
         else:
-            iterate_over = (
-                (os.path.basename(fp).split(".")[0], fp)
-                for fp in sorted(glob.glob(os.path.join(seqs.path, "*.fasta")))
+            # we got contigs or MAGs
+            def view_to_paths(view):
+                path = view[0]
```

Comment:
This adds a tiny amount of checking, and makes it a little more clear that the input is a tuple, rather than a view which has some kind of get-item maybe?
```suggestion
            def view_to_paths(arg):
                relpath, _ = arg
```